cmake/partition_manager.cmake

```diff
@@ -6,29 +6,60 @@
 
 if(FIRST_BOILERPLATE_EXECUTION)
   get_property(
-    partition_manager_config_files
+    images
     GLOBAL PROPERTY
-    PARTITION_MANAGER_CONFIG_FILES
+    PM_IMAGES
```

Comment:
Where is this populated? Is there a missing PR in `nrf` which populates this?

Reply:
Up now